mqtt_esp8266_rgb/mqtt_esp8266_rgb.ino

```diff
@@ -428,6 +468,27 @@ void loop() {
       }
     }
   }
+
+  if (pulse) {
+    if ( millis() % pulseTickEveryXms == 0) {
+      pulseCurrentBrightness += pulseDir;
```

Comment:
Maybe I'm misunderstanding something here, but wouldn't this only trigger if we manage to hit this `if` in a matching millisecond? I don't think we can guarantee that the loop will happen regularly enough to trust that.

I'll give this a more thorough review in the next few days.

Reply:
Good point.. my tests (on a WeMos D1 mini with 80MHz) showed that without the `delay(1)` the pulsing was way too fast, and with it, one cycle was around 5 seconds (like specified in the config) and the animation looked nice and smooth.
Will look into a possibility to refactor this.